### PR TITLE
Feature/89

### LINKED ISSUE #89: Support for multiple numbers  & clientRequestId in voice sdk

### PR DESCRIPTION
**Pull request by crispinkoech**
This closes #89

### CODE REVIEW

File: lib/voice.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ Voice.prototype.call = function (params) {
                         }
                     };
                 }
-                if (!(/^\+\d{1,3}\d{3,}$/).test(value)) {
+                if (!(/^(\+\d{1,3}\d{3,})(,\+\d{1,3}\d{3,})*$/).test(value)) {
```
Comment (kevokevoh) [inline lib/voice.js:33]: Why are you duplicating the phone_number regex?

**Review by kevokevoh — COMMENTED** [review]
LGTM just a slight clarification @crispinkoech

File: lib/voice.js (hunk 1 above)
Comment (crispinkoech) [reply lib/voice.js:33]: It was quickest regex I could write to accept at least 1 phone number, i.e. `+25412345678` and `+25412345678,+25412345678`. There's probably a simpler way to write it, lemme check

Comment (crispinkoech) [reply lib/voice.js:33]: @kevokevoh can't find a simpler/shorter way to write that regex :grimacing:

Comment (crispinkoech) [reply lib/voice.js:33]: @aksalj that should work, will update this ASAP

File: README.md [hunk 2]
```diff
@@ -126,12 +126,11 @@ For more information, please read [http://docs.africastalking.com/airtime/sendin
   - `lastReceivedId`: "This is the id of the message that you last processed". Defaults to `0`
 
 
-- `createSubscription({ shortCode, keyword, phoneNumber, checkoutToken })`: Create a premium subscription
+- `createSubscription({ shortCode, keyword, phoneNumber })`: Create a premium subscription
 
   - `shortCode`: This is the premium short code mapped to your account. `REQUIRED`
   - `keyword`: A premium keyword under the above short code and mapped to your account. `REQUIRED`
   - `phoneNumber`:  The phone number to be subscribed. `REQUIRED`
-  - `checkoutToken`: This is a token used to validate the subscription request. `REQUIRED`
```
Comment (crispinkoech) [inline README.md:134]: @kevokevoh not sure why this still shows up on the diff; you'd made this commit yeah?

**Pull request merged**